### ISSUE TITLE
[question] where is hrnet_w18_small_model_v2.pth?

### ISSUE THREAD

**Issue by Freed-Wu** [SimonVandenhende/Multi-Task-Learning-PyTorch#17]
sorry for my disturbance.
in <https://github.com/HRNet/HRNet-Image-Classification/releases/tag/PretrainedWeights>, hrnet_w18_small_model_v2.pth doesn't exist.

There  only exist

- HRNet_W18_C_cosinelr_cutmix_300epoch.pth.tar
- HRNet_W18_C_pretrained.pth
- HRNet_W18_C_ssld_pretrained.pth

so which pth is right?

**Comment by ShehrozAttique** [comment]
> sorry for my disturbance. in https://github.com/HRNet/HRNet-Image-Classification/releases/tag/PretrainedWeights, hrnet_w18_small_model_v2.pth doesn't exist.
> 
> There only exist
> 
> * HRNet_W18_C_cosinelr_cutmix_300epoch.pth.tar
> * HRNet_W18_C_pretrained.pth
> * HRNet_W18_C_ssld_pretrained.pth
> 
> so which pth is right?

Did you find it?

**Comment by Freed-Wu** [comment]
No. It is a dead project. No response after the paper is accepted, By my observation.
